docs/custom-metric-filter.md

```diff
@@ -0,0 +1,195 @@
+# ACM Custom Metric Filtering
+
+TL;DR: This file provides guides over adjusting the custom metrics which Config Sync exports to Cloud Monitoring, and workarounds for enabling/disabling
+the custom 
+
+## Background
+
+When ACM is installed on a GKE cluster where
+[default service accont](http://cloud/kubernetes-engine/docs/tutorials/authenticating-to-cloud-platform)
+is available, or on a GKE cluster that has
+[workload identity](http://cloud/kubernetes-engine/docs/how-to/workload-identity)
+enabled and
+[IAM is correctly setup](https://cloud.google.com/anthos-config-management/docs/how-to/monitoring-config-sync#custom-monitoring),
+ACM will try to apply the Open Telemetry Collector config map that contains
+[Googlecloud exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter)
+to write metrics into Cloud Monitoring. The default configuration contains a
+bare minimum list of metrics, but some customers might want to configure
+different backends or fine tune the metric list for better monitoring solutions.
+This doc serves as a workaround for users to opt out of custom monitoring, or
+adjust metric filtering.
+
+Note: this playbook refers to Anthos Config Management (ACM), but the same
+configuration settings are applicable to Config Sync. Since the line between
+product and feature is blurry here, we're just referring to ACM for the
+remainder, but the content applies whether the customer is running Config Sync
+as a standalone or the larger ACM suite.
+
+Note: This workaround applies to ACM 1.8+, when custom monitoring was
+integrated.
+
+## The default pipeline configuration in the latest release
+
+Config Sync will check for the default credential in the environment when
+starting up. For a GKE cluster which always have a default service account,
+Config Sync will deploy the following Open Telemetry Collector configuration to
+enable the Google Cloud monitoring pipelines.
+
+A pipiline is always consist of:
+
+-   receivers: receives metrics from all reconcilers inside Config Sync
+-   processors: filters or transforms metrics before sending out
+-   exporters: various backend endpoints and their configurations
+
+Important configs are annotated by comments. Config Sync uses
+[filter processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/filterprocessor)
+to control the exlusive list of metrics that goes into each monitoring backend.
+
+Here is [the current configuration](https://github.com/tiffanny29631/kpt-config-sync/blob/main/pkg/metrics/otel.go#L38) of Open Telemetry Collector
+in ACM.
+
+```
```

Comment:
I don't think we really need to in-line this. Just link to it.

Reply:
Done